benchmark/misc/util-extend-vs-object-assign.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const common = require('../common.js');
+const util = require('util');
+const v8 = require('v8');
+
+const bench = common.createBenchmark(main, {
+  type: ['util._extend', 'Object.assign',
+         'util._extend', 'Object.assign'],
+  n: [10e4]
+});
+
+function main(params) {
```

Comment:
minor nit: it's probably better to keep with the same variable naming as other benchmarks, in other words, s/params/conf/